releases/release-1.29/release-notes/maps/pr-120595-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 120595
+releasenote:
+  text: Fixed attaching volumes after detach errors. Now volumes that failed to detach
+    are not treated as attached. Kubernetes will make sure they are fully attached
```

Comment:
The original statement is as follows:
```
Fixed attaching volumes after detach errors. Now volumes that failed to detach are not treated as attached, Kubernetes will make sure they are fully attached before they can be used by pods.
```
Are the words "before they can be used by pods." missing?